echopype/convert/set_groups_ek60.py

```diff
@@ -29,23 +29,14 @@ class SetGroupsEK60(SetGroupsBase):
     # the SONAR-netCDF4 v1 convention. For the time being, we are retaining the
     # infrastructure that adds this dimension, but updating the variables lists.
 
+    # 2023-07-21 note: We are also removing the ping_time dimension for parameters
```

Comment:
This comment seems unnecessary. Otherwise an analogous comment should be added to the EK80 and AZFP modules, and for the removal of the beam dimension.

Reply:
I added this because you added something like that just above this line. I was thinking this should actually be in set_groups_base. I'll move both comments to the leftover mechanism (which is entirely bypassed at this point).

Reply:
Did I? Sigh. It may have been a temporary note to self that I forgot to remove. I agree that it makes more sense to have something like that in `set_groups_base` or elsewhere. Thanks.